src/jbi/models.py

```diff
@@ -17,6 +17,7 @@ class Action(YamlModel):
 
     action: str = "src.jbi.whiteboard_actions.default"
     enabled: bool = False
+    description: str
     allow_private: bool = False
     parameters: dict = {}
```

Comment:
```suggestion
    action: str = "src.jbi.whiteboard_actions.default"
    allow_private: bool = False    
    contact: str
    description: str
    enabled: bool = False
    parameters: dict = {}
```

Reply:
Suggestion: order params as expected in config; add `contact` field.

Reply:
https://github.com/mozilla/jira-bugzilla-integration/pull/84/commits/f0d07aca4bba3b8a97f3156892a88b2985d039d6